home.php

```diff
@@ -169,14 +169,14 @@ function getEventsByStudent($tournamentID, $studentID, $dateTournament)
 		$output .= "<p><a type='button' class='btn btn-primary' href='#news-edit'><span class='bi bi-edit'></span> Edit News</button></a></p>";
 	}
 
+	$output .= "<hr><h2>My Profile</h2>";
 	$output .= '<img src="'.$_SESSION['userData']['picture'].'">';
-	$output .= '<p><b>Name:</b> '.$_SESSION['userData']['first_name'].' '.$_SESSION['userData']['last_name'].'</p>';
-	$output .= '<p><b>Email:</b> '.$_SESSION['userData']['email'].'</p>';
+	$output .= '<p><b>Name:</b> '.$_SESSION['userData']['first_name'].' '.$_SESSION['userData']['last_name'].'<br><b>Email:</b> '.$_SESSION['userData']['email'].'</p>';
```

Comment:
I usually avoid line breaks inside paragraphs.  Was there a reason that you decided not to just end a paragraph and start a new paragraph.

Reply:
The embedded speadsheet for meeting times should just be put in the news that is stored in the db.  The front page is not meant to contain information that may need to be changed rapidly.  
During our meeting let's review the addition of the collapsibles.  I think these are great.